templates/pre_reqs.sh

```diff
@@ -106,4 +106,4 @@ sed -i "s|# ingressVIP: 10.0.0.2|ingressVIP: $INGRESS_VIP|g" $cluster_config
 sed -i "s|      - address: <Machine 1 IP>|$cp_string|g" $cluster_config
 sed -i "s|  - address: <Machine 2 IP>|$worker_string|g" $cluster_config
 sed -i "s|- 10.96.0.0/12|- 172.31.0.0/16|g" $cluster_config
-sed -i "s|- 192.168.0.0/16|- 172.30.0.0/16|g" $cluster_config
+sed -i "s|- 192.168.0.0/16|- 172.30.0.0/16|g" $cluster_config
```

Comment:
Why are we removing the new line at the end of this file? Isn't leaving a new line best practice?